.github/workflows/ci.yml

```diff
@@ -43,3 +43,100 @@ jobs:
+        connector-name:
+          - test-connector
+          - fluvio-syslog
+        rust: [stable]
+    env:
+      RUSTV: ${{ matrix.rust }}
+      TARGET: ${{ matrix.rust-target }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set RELEASE mode
+        #if: github.event_name == 'push' && github.ref == 'refs/heads/staging'
+        shell: bash
+        run: |
+          echo "RELEASE=true" | tee -a $GITHUB_ENV
+          echo "RELEASE_NAME=release" | tee -a $GITHUB_ENV
+          echo "RUST_BIN_DIR=target/${{ matrix.rust-target }}/release" | tee -a $GITHUB_ENV
+      - name: Install Rust stable
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: ${{ matrix.rust }} 
+          override: true
+          target: ${{ matrix.rust-target }}
+      - name: Install system toolchain - x86_64-unknown-linux-musl'
+        if: ${{ matrix.rust-target == 'x86_64-unknown-linux-musl' }}
+        run: |
+          sudo apt update
+          sudo apt install -y musl-tools
+
+      #- name: Install system toolchain - aarch64-unknown-linux-musl
+      #  if: ${{ matrix.rust-target == 'aarch64-unknown-linux-musl' }}
+      #  run: |
+      #    apt update
+      #    apt install -y gcc-aarch64-linux-gnu 
+
+      - name: Build
+        env:
+          CONNECTOR_NAME: ${{ matrix.connector-name }}
+        run: make build
+
+      # Upload artifacts
+      - name: Upload artifact - ${{ matrix.binary }}
+        uses: actions/upload-artifact@v2
+        with:
+          name: ${{ matrix.connector-name }}-${{ matrix.rust-target }}
+          path: ${{ env.RUST_BIN_DIR }}/${{ matrix.connector-name }}
+
+  build_containers:
+    name: Build containers 
+    needs: build_connector_bins 
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        rust-target:
+          - x86_64-unknown-linux-musl
+          #- aarch64-unknown-linux-musl
+        include:
+          - connector-name: test-connector
+            image-name: infinyon/fluvio-connect-test-connector
+          - connector-name: fluvio-syslog
+            image-name: infinyon/fluvio-connect-syslog
+        rust: [stable]
+    env:
+      RUSTV: ${{ matrix.rust }}
+      TARGET: ${{ matrix.rust-target }}
+    steps:
+      - uses: actions/checkout@v2
+
+      # Download artifacts from build
+      - name: Download artifact - fluvio-run
+        uses: actions/download-artifact@v2
+        with:
+          name: ${{ matrix.connector-name }}-${{ matrix.rust-target }}
+          path: container-build 
+
+      #- name: Install system toolchain - aarch64-unknown-linux-musl
+      #  if: ${{ matrix.rust-target == 'aarch64-unknown-linux-musl' }}
+      #  run: |
+      #    apt update
+      #    apt install -y gcc-aarch64-linux-gnu 
+
+      - name: Build containers
+        run: make IMAGE_NAME=${{ matrix.image-name }} CONNECTOR_NAME=${{ matrix.connector-name }} official-containers
```

Comment:
Oh, you fancy. Parallel build.

Reply:
Yeah! But in development you can just run `make official-containers` and you'll build the test-connector by default